packages/terminal/src/browser/terminal-widget.ts

```diff
@@ -54,17 +55,19 @@ export class TerminalWidget extends BaseWidget implements StatefulWidget {
 
     private terminalId: number | undefined;
     private term: Xterm.Terminal;
-    private cols: number = 80;
-    private rows: number = 40;
+    private cols: number;
+    private rows: number;
     private endpoint: Endpoint;
     protected restored = false;
     protected closeOnDispose = true;
-    protected waitForStarted: Promise<void>;
-    protected started: Function;
     protected openAfterShow = false;
     protected isOpeningTerm = false;
     protected isTermOpen = false;
 
+    protected waitForStarted = new Deferred<void>();
```

Comment:
I do not see that we wait for this promise anywhere. Should it be added in the ```Promise.all()``` in ```doResize()```?

Reply:
No it is no longer neccessary. I will remove it thanks.